app/styles/live.module.css

```diff
@@ -0,0 +1,14 @@
+#live {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 16px;
```

Comment:
Add `font-size` in rem. This is already followed in the codebase, you will find everywhere `font-size` in rems.

Reply:
Yeah👍